src/adapter/src/error.rs

```diff
@@ -442,12 +449,15 @@ impl AdapterError {
             AdapterError::Unsupported(..) => SqlState::FEATURE_NOT_SUPPORTED,
             AdapterError::Unstructured(_) => SqlState::INTERNAL_ERROR,
             AdapterError::UntargetedLogRead { .. } => SqlState::FEATURE_NOT_SUPPORTED,
+            AdapterError::DDLTransactionRace => SqlState::T_R_SERIALIZATION_FAILURE,
+            AdapterError::DDLTransactionDryRun => SqlState::T_R_SERIALIZATION_FAILURE,
```

Comment:
Uncovered

Reply:
This is expected to be uncovered, since this error should never get returned to the user